pkgs/development/python-modules/staticjinja/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, docopt
+, easywatch
+, jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "staticjinja";
+  version = "0.3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1mxv7yy35657mfxx9xhbzihh10m5lb29fmscfh9q455zd4ikr032";
+  };
+
```

Comment:
There is a test on github https://github.com/Ceasar/staticjinja/blob/master/test_staticjinja.py
but it's not in the pypi tarball.

Not too extensive so I don't think it matters.

So we need `doCheck = false;` here as well.

Reply:
And add a comment with the reason.